config.go

```diff
@@ -62,15 +69,20 @@ func parseConfig(args []string) (conf *Config, err error) {
 	fs.Var(&retry, "retry", "connection retry interval (default 30s)")
 	fs.Var(&flush, "flush", "buffer flush interval (default 500ms)")
 	fs.Var(&apiKey, "apikey", "apikey or path to apikey file")
+	fs.Var(&qfilter, "filter_qname", "suppress responses to queries under domain")
 	fs.UintVar(&channel, "channel", 0, "channel to upload dnstap data")
+	fs.IntVar(&mtu, "mtu", 0, "UDP output buffer size")
```

Comment:
Should the minimum 1280 also be mentioned here?

Reply:
Fixed, sort of, by making the default explicitly `nmsg.EtherContainerSize`